demisto_sdk/commands/format/update_layout.py

```diff
@@ -211,3 +219,44 @@ def layoutscontainer__arguments_to_remove(self):
                 second_level_args[kind] = set(self.data[kind].keys()) - set(kind_schema)
 
         return first_level_args, second_level_args
+
+    def remove_copy_and_dev_suffixes_from_layoutscontainer(self):
+        if name := self.data.get('name'):
+            self.data['name'] = remove_copy_and_dev_suffixes_from_str(name)
+
+        container = None
+        for kind in LAYOUTS_CONTAINER_CHECK_SCRIPTS:
+            if self.data.get(kind):
+                container = self.data.get(kind)
+                break
+        if container:
+            for tab in container.get('tabs'):
+                if tab.get('sections'):
```

Comment:
this indentation level can be removed, as the following `for` will not run if `not tab.get('sections')`

Reply:
done